torch/distributed/algorithms/ddp_comm_hooks/powerSGD_hook.py

```diff
@@ -264,6 +266,31 @@ def __init__(
         # Turn on if compression / decompression computation is a bottleneck.
         self.batch_tensors_with_same_shape = batch_tensors_with_same_shape
 
+    def __getstate__(self):
+        r"""
+        Returns a ``PowerSGDState`` which will be pickled and saved.
```

Comment:
nit: I think the return type is not `PowerSGDState` but rather `Dict[str, Any]`?

Reply:
This is a valid point. Thanks for correcting. I've changed it in the updated PR.